translated-pairs/en-replace.ru.pairs

```diff
@@ -300,41 +300,41 @@ Total number of evaluated code phrases
 
 #### left half of score summary
 Finished in:
-Finished in:
+Закончено за:
 
 Solution size:
-Solution size:
+Размер решения:
 
 Run size:
-Run size:
+Размер выполнения:
 
 #### right half `{}` is replaced by the score
 {} ms
-{} ms
+{} мс
 
 {} phrases
-{} phrases
+{} (фразы)
 
 #### Graph button
 Show outliers
-Show outliers
+Подробная статистика
 
 Hide outliers
-Hide outliers
+Компактный вид
 
 #### Graph title
 Time
-Time
+Время
 
 Solution size
-Solution size
+Размер решения
 
 Run size
-Run size
+Размер выполнения
 
 #### Label for the vertical line marking a player's best/lowest score
 Best
-Best
+Лучш.
```

Comment:
Should this have a full stop `.`, it doesn't seem right _(though of course I don't understand Russian!)_

Reply:
Yeah, it looks terrible, but we have gender inflexions in Russian, so Time is "Лучшее", Size is "Лучший" and so on. I could use "Лучший результат" (The best result), but it's too long(

There's another way:
```
#### Graph title
Time
Время

Solution size
Кол-во фраз //"Кол-во" is a popular shrinkage for "Количество" - Quantity of phrases

Run size
Общее кол-во фраз //Total quantity of phrases

Best
Лучшее 
```

It fits anything.

Reply:
Right interesting, just to be clear we're talking about the word marking your best score.
![](https://user-images.githubusercontent.com/2331607/70828060-7de20400-1de2-11ea-8493-3d55e54bf27d.png)

Are you saying it would be better to have different words for each graph? For now it's probably best to go with the least bad word that fits everywhere.

Reply:
Best score, yep. So do I have to commit a new one?